Geotools oddity; seeinge a compiler failure for this PR:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project viewer: Compilation failure
[ERROR] /home/travis/build/flamingo-geocms/flamingo/viewer/src/main/java/nl/b3p/viewer/util/ValidFilterExtractor.java:[76,25] cannot access org.opengis.filter.NativeFilter
[ERROR]   class file for org.opengis.filter.NativeFilter not found
```
https://travis-ci.org/flamingo-geocms/flamingo/jobs/446883218#L2268-L2269


However it seems that NativeFilter was not added until Geotools 20 (https://github.com/geotools/geotools/commit/aecfbcaff5e01f3d35aaf38826de43b051cd4603#diff-8486007c73d60fa0534450ca20f88f39)

--- a/.travis.yml
+++ b/.travis.yml
@@ -66,15 +66,12 @@ after_success:
 jdk:
   - oraclejdk8
   - openjdk8
-  - oraclejdk9
 
 os:
   - linux
 
 matrix:
   fast_finish: true
-  allow_failures:
-    - jdk: oraclejdk9
 
 cache:
   directories:

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -48,7 +48,7 @@ timestamps {
             }
 
             stage('OWASP Dependency Check') {
-                sh "mvn org.owasp:dependency-check-maven:aggregate -Dformat=XML -DsuppressionFile=./.mvn/owasp-suppression.xml"
+                sh "mvn org.owasp:dependency-check-maven:aggregate -DskipSystemScope=true -DnodeAuditAnalyzerEnabled=false -DnodeAnalyzerEnabled=false -Dformat=XML -DsuppressionFile=./.mvn/owasp-suppression.xml"
 
                 dependencyCheckPublisher canComputeNew: false, defaultEncoding: '', healthy: '85', pattern: '**/dependency-check-report.xml', shouldDetectModules: true, unHealthy: ''
             }

--- a/pom.xml
+++ b/pom.xml
@@ -7,9 +7,6 @@
     <packaging>pom</packaging>
     <name>flamingo-mc</name>
     <url>http://github.com/flamingo-mc</url>
-    <prerequisites>
-        <maven>3.2.5</maven>
-    </prerequisites>
     <scm>
         <connection>scm:git:git@github.com:flamingo-mc/flamingo.git</connection>
         <developerConnection>scm:git:git@github.com:flamingo-mc/flamingo.git</developerConnection>
@@ -20,12 +17,12 @@
         <repository>
             <id>repo.b3p.nl</id>
             <name>B3Partners releases repository</name>
-            <url>http://repo.b3p.nl/nexus/content/repositories/releases/</url>
+            <url>https://repo.b3p.nl/nexus/repository/releases/</url>
         </repository>
         <snapshotRepository>
             <id>repo.b3p.nl</id>
             <name>B3Partners snapshots repository</name>
-            <url>http://repo.b3p.nl/nexus/content/repositories/snapshots/</url>
+            <url>https://repo.b3p.nl/nexus/repository/snapshots/</url>
         </snapshotRepository>
     </distributionManagement>
     <licenses>
@@ -495,7 +492,7 @@
         <repository>
             <id>B3Partners</id>
             <name>Releases hosted by B3Partners</name>
-            <url>http://repo.b3p.nl/nexus/content/groups/public/</url>
+            <url>https://repo.b3p.nl/nexus/repository/public/</url>
         </repository>
     </repositories>
     <build>
@@ -805,25 +802,25 @@
             <name>Matthijs Laan</name>
             <email>matthijslaan@b3partners.nl</email>
             <organization>B3Partners B.V.</organization>
-            <organizationUrl>http://www.b3partners.nl</organizationUrl>
+            <organizationUrl>https://www.b3partners.nl</organizationUrl>
         </developer>
         <developer>
             <name>Meine Toonen</name>
             <email>meinetoonen@b3partners.nl</email>
             <organization>B3Partners B.V.</organization>
-            <organizationUrl>http://www.b3partners.nl</organizationUrl>
+            <organizationUrl>https://www.b3partners.nl</organizationUrl>
         </developer>
         <developer>
             <name>Roy Braam</name>
             <email>roybraam@b3partners.nl</email>
             <organization>B3Partners B.V.</organization>
-            <organizationUrl>http://www.b3partners.nl</organizationUrl>
+            <organizationUrl>https://www.b3partners.nl</organizationUrl>
         </developer>
         <developer>
             <name>Mark Prins</name>
             <email>markprins@b3partners.nl</email>
             <organization>B3Partners B.V.</organization>
-            <organizationUrl>http://www.b3partners.nl</organizationUrl>
+            <organizationUrl>https://www.b3partners.nl</organizationUrl>
         </developer>
     </developers>
 </project>

--- a/pom.xml
+++ b/pom.xml
@@ -42,6 +42,10 @@
         <maven.compiler.target>${java.version}</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <!--
+            when upgrading geotools also check b3p-commons-csw which has some
+            geotools transitive dependencies
+        -->
         <geotools.version>19.3</geotools.version>
         <powermock.version>1.7.4</powermock.version>
         <hsqldb.version>2.4.1</hsqldb.version>
@@ -86,6 +90,7 @@
                 <artifactId>gt-render</artifactId>
                 <version>${geotools.version}</version>
             </dependency>
+
             <dependency>
                 <groupId>org.json</groupId>
                 <artifactId>json</artifactId>
@@ -150,7 +155,7 @@
             <dependency>
                 <groupId>nl.b3p</groupId>
                 <artifactId>b3p-commons-csw</artifactId>
-                <version>5.0.5</version>
+                <version>5.0.6</version>
                 <exclusions>
                     <exclusion>
                         <groupId>javax.persistence</groupId>
@@ -595,7 +600,7 @@
                 <plugin>
                     <groupId>org.owasp</groupId>
                     <artifactId>dependency-check-maven</artifactId>
-                    <version>3.3.2</version>
+                    <version>3.3.4</version>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.tomcat.maven</groupId>

